ERROR in ./node_modules/readable-web-to-node-stream/lib/index.js Module not found

With Angular@10.1.6 and file-type@16.0.0 and get the following error in console during startup:
`ERROR in ./node_modules/readable-web-to-node-stream/lib/index.js
Module not found: Error: Can't resolve 'stream' in `

The file exists in node_modules directory, nothing seems wrong.
But using an older version helps. The last version, that works without this error, is file-type@12.4.2

Is there any known issue?
Or what am I doing wrong?

--- a/package.json
+++ b/package.json
@@ -190,7 +190,7 @@
 		"xo": "^0.25.3"
 	},
 	"dependencies": {
-		"readable-web-to-node-stream": "^2.0.0",
+		"readable-web-to-node-stream": "^3.0.0",
 		"strtok3": "^6.0.3",
 		"token-types": "^2.0.0",
 		"typedarray-to-buffer": "^3.1.5"

